src/framework/constraints.ml

```diff
@@ -1382,10 +1382,10 @@ struct
     nh
 
   let compare (name1, name2) vh1 vh2 =
-    Printf.printf "\nComparing nodes precision of %s (left) with %s (right):\n" name1 name2;
+    let verbose = get_bool "dbg.verbose" in
+    if verbose then Printf.printf "\nComparing nodes precision of %s (left) with %s (right):\n" name1 name2;
     let vh1' = join_contexts vh1 in
     let vh2' = join_contexts vh2 in
-    let verbose = get_bool "dbg.compare_runs.diff" in
```

Comment:
Are these changes really necessary for the scripts to work?

Reply:
Without it, the output comparison log files get huge and very difficult to read or only open. My text editor always crashed whenever I tried which is very annoying. I was also worried, that it might slow down the experiments just by the number of lines that are written to the log.